test/blockchain/vbk_blockchain_util_test.cpp

```diff
@@ -0,0 +1,191 @@
+#include <gtest/gtest.h>
+
+#include <memory>
+
+#include "veriblock/arith_uint256.hpp"
+#include "veriblock/blockchain/block_index.hpp"
+#include "veriblock/blockchain/vbk_blockchain_util.hpp"
+#include "veriblock/blockchain/vbk_chain_params.hpp"
+
+using namespace VeriBlock;
+
+static std::vector<BlockIndex<VbkBlock>> getChain(int32_t deltaTime,
+                                                  int32_t deltaTime_change,
+                                                  const uint32_t& difficulty,
+                                                  const uint32_t& chainlength) {
+  BlockIndex<VbkBlock> blockIndex;
+  blockIndex.header.height = 1;
+  blockIndex.header.timestamp = 10000;
+  blockIndex.header.difficulty = difficulty;
+  blockIndex.height = blockIndex.header.height;
+  blockIndex.pprev = nullptr;
+
+  std::vector<BlockIndex<VbkBlock>> chain(chainlength);
+  chain[0] = blockIndex;
```

Comment:
If chainlength is zero an exception will be thrown here. Put an assert to make sure chainlength is not zero.

Reply:
fixed